extensions/amp-bind/0.1/amp-state.js

```diff
@@ -16,13 +16,24 @@
 
 import {bindForDoc} from '../../../src/services';
 import {fetchBatchedJsonFor} from '../../../src/batched-json';
+import {getMode} from '../../../src/mode';
 import {isBindEnabledFor} from './bind-impl';
 import {isJsonScriptTag} from '../../../src/dom';
 import {toggle} from '../../../src/style';
 import {tryParseJson} from '../../../src/json';
 import {user} from '../../../src/log';
 
 export class AmpState extends AMP.BaseElement {
+
+  /** @param {!AmpElement} element */
+  constructor(element) {
+    super(element);
+
+    /** @visibleForTesting {?Promise} */
+    this.updateStatePromise = null;
```

Comment:
Did I miss doing that somewhere, or do you mean setting it to null here should only be done during testing as well?